src/main/java/com/codeborne/selenide/selector/ByLabel.java

```diff
@@ -8,21 +9,23 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class ByTextCaseInsensitive extends OptimizedBy {
-  private static final JavaScript js = new JavaScript("find-elements-by-text-case-insensitive.js");
+public class ByLabel extends OptimizedBy {
+  private static final JavaScript js = new JavaScript("find-elements-by-label.js");
   private final String elementText;
+  private final TextMatchOptions options;
 
-  public ByTextCaseInsensitive(String elementText) {
+  public ByLabel(String elementText, TextMatchOptions options) {
     this.elementText = elementText;
+    this.options = options;
   }
 
   @Override
   protected List<WebElement> findElements(SearchContext context, int limit) {
-    return requireNonNull(js.execute(context, js.node(context), elementText, limit));
+    return requireNonNull(js.execute(context, js.node(context), elementText, options.toMap(), limit));
   }
 
   @Override
   public String toString() {
-    return "by text (case-insensitive): " + elementText;
+    return "by label: " + elementText;
```

Comment:
Do you want to include options? 

`"by label (%s): \"%s\"".formatted(options, elementText);`

Reply:
Good point, fixed.